test/lib/util.js

```diff
@@ -291,6 +291,15 @@ function makeEncodingParameters(options) {
   return encodingParameters;
 }
 
+/**
+ * Wait for some time.
+ * @param {number} ms - time to wait in milliseconds.
+ * @returns {Promise<void>}
+ */
+function waitFor(ms) {
```

Comment:
this should be named `waitForSometime` to avoid potential conflict with `waitFor` in V2 that takes promise as a parameter